go.mod

```diff
@@ -23,11 +23,11 @@ require (
 	github.com/prometheus/common v0.44.0 // indirect
 	github.com/prometheus/procfs v0.10.1 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
-	golang.org/x/net v0.12.0 // indirect
+	golang.org/x/net v0.13.0 // indirect
```

Comment:
In line 8, github.com/kubernetes-csi/csi-lib-utils v0.15.0-rc0 we still need to update to v0.15.0 when it's ready

Reply:
Updated this to `csi-lib-utils: v0.14.0`. External-attacher: PR [#475](https://github.com/kubernetes-csi/external-attacher/pull/475); External-provisioner: [#999](https://github.com/kubernetes-csi/external-provisioner/pull/999)